packages/accounts-controller/package.json

```diff
@@ -61,6 +61,7 @@
     "@types/readable-stream": "^2.3.0",
     "jest": "^27.5.1",
     "ts-jest": "^27.1.4",
+    "tsup": "^8.0.2",
```

Comment:
Added to each package as well as the root package so that it's possible to run `yarn workspace <package> run build`, for testing purposes.